src/main/java/core/basesyntax/FileWork.java

```diff
@@ -6,11 +6,12 @@
 import java.util.Arrays;
 
 public class FileWork {
```

Comment:
what about these https://github.com/mate-academy/jv-read-from-file/pull/239#discussion_r607730013 & https://github.com/mate-academy/jv-read-from-file/pull/239#discussion_r607728796

Reply:
Without this check i have failed 1 test(

Reply:
ok. I see it now) all good